inherited_resources.gemspec

```diff
@@ -14,7 +15,9 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = "inherited_resources"
 
-  s.files         = Dir["app/**/*", "lib/**/*", "README.md", "MIT-LICENSE"]
+  s.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
```

Comment:
I'm aware bundler's default is to shell out to `git`. My preferred solution though is the current one, since I find it more explicit and it doesn't cause errror messages when building the gem on systems without `git` installed (like very light docker images). Bundler's default approach is meant for non experienced users creating gems for the first time, because it prevents them from accidentally shipping generated (not source controlled) files with the gem.

Reply:
I have no issue with that too. I will leave it as it was then. Thanks!